ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master__edge.yaml

```diff
@@ -74,7 +74,7 @@ test_binary_build_commands: |
   mkdir /.cache && chmod 775 -R /.cache ${GOPATH}
 tests:
 - as: publish-multi-arch-images
-  postsubmit: true
+  postsubmit: false
```

Comment:
be aware those are temporary changes just to enable testing in this PR. so it should not be merged